lab_3_generate_by_ngrams/main.py

```diff
@@ -121,6 +168,15 @@ def decode(self, encoded_corpus: tuple[int, ...]) -> Optional[str]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(encoded_corpus, tuple) or encoded_corpus == ():
+            return None
+        decoded_function = self._decode(encoded_corpus)
```

Comment:
I have questions for your variable names here: these aren't functions, are they?

Reply:
i meant only the fact that in this variable i will save the work of the _decode function
Should i check the names?